### PR TITLE
NOSTORY: Fixed items usage error

### PR DESCRIPTION
**Pull request by patrickmukumbu**
# Description

This PR has been created to fix the errors resulting from the usage of table items.

## Features/Fixes/Patches/Changes list:

Please check the [CHANGELOG.md](https://github.com/boldlink/terraform-aws-dynamodb/blob/fixes/ddb-item/CHANGELOG.md)

### PR Owners Checklist:
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you checked there aren't open [Pull Requests](../../../pulls) of upstream or parallel code that will cause conflicts?
* [x] Have you updated the `CHANGELOG.md`?
* [x] Have you updated the `README.md`(s)?
* [ ] OWNER: Does your submission pass?
    * [x] Pre-commit (attach logs/print screen to this PR)
    * [x] Checkov/terrascan (attach logs/print screen to this PR)
    * [x] Terraform resources examples build/destroy successfully with Terraform 0.14.11 minimum?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully run tests with your changes locally?
* [ ] First PR? Have you deleted `## How to use this template...` from the root `.README.md`?

### PR Reviewers Checklists?
* [ ] Are all your comments/changes resolved?
* [ ] Are you happy with the OWNER checklists and additional information provided?
* [ ] Does your review tests pass?
    * [ ] Terraform resources examples build/destroy successfully with Terraform 0.14.11 minimum?

### Checkov Analysis Result
[dynamodb-checkov.txt](https://github.com/boldlink/terraform-aws-dynamodb/files/8831758/dynamodb-checkov.txt)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,8 +13,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add: Usage of local secondary index in example(s)
 - Feature: Modify main module to ensure values in gsi/lsi match autoscaling values to avoid inconsistencies.
 
-## [1.1.0] - 2022-05-17
+## [1.1.1] - 2022-06-03
 
+### Fixed
+- Fix: Error(s) that arise using table item(s).
+
+### Added
```
Comment (hugoalmeida264) [inline CHANGELOG.md]: No need to separate them on tho sections, we can have just one titles `Changes`

File: examples/table_items/main.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      name = "exampleHashKey"
+      type = "S"
+    }
+  ]
+
+  ### **NOTE**: aws_dynamodb_table_item is **NOT** meant to be used for managing large amounts of data in your table,
+  ### it is not designed to scale. You should perform regular backups of all data in the table
+
+  table_items = {
+    item = <<ITEM
```
Comment (hugoalmeida264) [inline examples/table_items/main.tf]: Use `jsoncode()`

**Review by hugoalmeida264 — CHANGES_REQUESTED** [review]
Hi Patrick - see my comments

**Comment by patrickmukumbu** [comment]
Corrected @hugoalmeida264 . Please have a look

**Review by hugoalmeida264 — APPROVED** [review]

**Review by j0hnmwangi — APPROVED** [review]

**Pull request merged**